app/vmui/packages/vmui/src/utils/time.ts

```diff
@@ -26,6 +26,22 @@ const shortDurations = supportedDurations.map(d => d.short);
 
 export const roundToMilliseconds = (num: number): number => Math.round(num*1000)/1000;
 
+const roundStep = (step: number) => {
+  if (step >= 100) {
+    return Math.round(step) - (Math.round(step)%10); // step 10
```

Comment:
we can save Math.round(step) into const value and use it in the next calculations